plugin/src/main/java/org/opensearch/ml/model/MLTask.java

```diff
@@ -63,6 +65,7 @@
     @Setter
     private String error;
     private User user; // TODO: support document level access control later
+    private boolean async;
 
     @Builder
     public MLTask(
```

Comment:
We can consider moving this class to the ml-common in the next PR.

Reply:
Yes, will send out a separate refactoring PR